openverse_catalog/dags/common/loader/sql.py

```diff
@@ -326,7 +325,7 @@ def _get_malformed_row_in_file(error_msg):
 
 
 def _delete_malformed_row_in_file(tsv_file_name, line_number):
-    with open(tsv_file_name, "r") as read_obj:
+    with open(tsv_file_name) as read_obj:
```

Comment:
Did `pyugrade` catch this? Interesting!

Reply:
It does! It removes "redundant file modes". The full list of transformations it does is pretty interesting. I was sceptical at first but as I kept reading them they make sense. All the ones that I was most sceptical of also had caveats that covered the edge cases I was concerned about (especially format string transformations).